cluster/gce/configure-vm.sh

```diff
@@ -254,6 +254,13 @@ function create-salt-auth() {
   kubelet_auth_file="/srv/salt-overlay/salt/kubelet/kubernetes_auth"
   (umask 077;
     echo "{\"BearerToken\": \"${KUBELET_TOKEN}\", \"Insecure\": true }" > "${kubelet_auth_file}")
+
+  # Generate tokens for other "service accounts".  Append to known_tokens.
+  for a in "system:scheduler" "system:controller_manager" "system:logging" "system:monitoring" "system:dns"
+  do
+    token=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+    echo "${token},$a,$a" >> "${known_tokens_file}"
```

Comment:
Nit: I try to follow ${foo} for all variables. (so `${a}`. Or `${account}`). And below.